api/managementpb/checks.proto

```diff
@@ -10,8 +10,42 @@ message StartSecurityChecksRequest {}
 
 message StartSecurityChecksResponse {}
 
+message GetSecurityCheckResultsRequest {}
+
+// Severity represents severity level of the check result.
+enum Severity {
+  SEVERITY_INVALID = 0;
+  SEVERITY_EMERGENCY = 1;
+  SEVERITY_ALERT = 2;
+  SEVERITY_CRITICAL = 3;
+  SEVERITY_ERROR = 4;
+  SEVERITY_WARNING = 5;
+  SEVERITY_NOTICE = 6;
+  SEVERITY_INFO = 7;
+  SEVERITY_DEBUG = 8;
+}
+
+// STTCheckResult represents the check result returned from pmm-managed after running the check.
+message STTCheckResult {
```

Comment:
I think better to name it `SecurityCheckResult` to be consistent with `GetSecurityCheckResultsRequest`, `GetSecurityCheckResultsResponse`.

Reply:
+1